app/views/profiles/show.html.erb

```diff
@@ -17,6 +17,13 @@
 <%= link_to t('.edit', :default => t("helpers.links.edit")),
               edit_profile_path(@profile), :class => 'btn btn-default' %>
 
+<%= link_to t('.destroy', :default => t("helpers.links.destroy")),
+              profile_path(@profile),
+              :method => 'delete',
+              :data => { :confirm => t('.confirm', :default => t("helpers.links.confirm", :default => 'Are you sure?')) },
+              :class => 'btn btn-danger' %>
```

Comment:
You're right, looks like the automatic merge messed up. This was removed from the dev [last week](https://github.com/hpi-swt2/workshop-portal/commit/48418a55dba13ec3e101caa2c8e28c1193ad9491), but since you had modified the line it was kept. It can be deleted now :slightly_smiling_face: